frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx

```diff
@@ -80,10 +42,15 @@ const SelectCapacityAndNodes: React.FC<SelectCapacityAndNodesProps> = ({
 }) => {
   const { t } = useTranslation();
 
+  React.useEffect(() => {
+    if (!capacity) dispatch({ type: 'capacityAndNodes/capacity', payload: '2Ti' });
```

Comment:
It was previously but there are two flows - local devices and internal.
In case of local devices we dont want 2Ti. hence initializing states separately in those components

Reply:
ack